.gitignore

```diff
@@ -1,2 +1,5 @@
 todo
-environment.py
\ No newline at end of file
+environment.py
```

Comment:
If the purpose of this entry is to include local environment variables, use `.env` as the name of the file instead. It's the convention and more easy to understand, plus the file is hidden by default